recipes/openimageio/all/conanfile.py

```diff
@@ -63,260 +64,248 @@ class OpenImageIOConan(ConanFile):
-        cmake.definitions["USE_OPENCV"] = self.options.with_opencv
-        cmake.definitions["USE_TBB"] = self.options.with_tbb
-        cmake.definitions["USE_DCMTK"] = self.options.with_dicom
-        cmake.definitions["USE_FFMPEG"] = self.options.with_ffmpeg
-        cmake.definitions["USE_FIELD3D"] = False
-        cmake.definitions["USE_GIF"] = self.options.with_giflib
-        cmake.definitions["USE_LIBHEIF"] = self.options.with_libheif
-        cmake.definitions["USE_LIBRAW"] = self.options.with_raw
-        cmake.definitions["USE_OPENVDB"] = self.options.with_openvdb
-        cmake.definitions["USE_PTEX"] = self.options.with_ptex
-        cmake.definitions["USE_R3DSDK"] = False
-        cmake.definitions["USE_NUKE"] = False
-        cmake.definitions["USE_OPENGL"] = False
-        cmake.definitions["USE_QT"] = False
-        cmake.definitions["USE_LIBPNG"] = self.options.with_libpng
-        cmake.definitions["USE_FREETYPE"] = self.options.with_freetype
-        cmake.definitions["USE_LIBWEBP"] = self.options.with_libwebp
-        cmake.definitions["USE_OPENJPEG"] = self.options.with_openjpeg
-
-        if self.options.with_openvdb:
-            cmake.definitions["CMAKE_CXX_STANDARD"] = 14
+        tc.variables["USE_HDF5"] = self.options.with_hdf5
+        tc.variables["USE_OPENCOLORIO"] = self.options.with_opencolorio
+        tc.variables["USE_OPENCV"] = self.options.with_opencv
+        tc.variables["USE_TBB"] = self.options.with_tbb
+        tc.variables["USE_DCMTK"] = self.options.with_dicom
+        tc.variables["USE_FFMPEG"] = self.options.with_ffmpeg
+        tc.variables["USE_FIELD3D"] = False
+        tc.variables["USE_GIF"] = self.options.with_giflib
+        tc.variables["USE_LIBHEIF"] = self.options.with_libheif
+        tc.variables["USE_LIBRAW"] = self.options.with_raw
+        tc.variables["USE_OPENVDB"] = self.options.with_openvdb
+        tc.variables["USE_PTEX"] = self.options.with_ptex
+        tc.variables["USE_R3DSDK"] = False
+        tc.variables["USE_NUKE"] = False
+        tc.variables["USE_OPENGL"] = False
+        tc.variables["USE_QT"] = False
+        tc.variables["USE_LIBPNG"] = self.options.with_libpng
+        tc.variables["USE_FREETYPE"] = self.options.with_freetype
+        tc.variables["USE_LIBWEBP"] = self.options.with_libwebp
+        tc.variables["USE_OPENJPEG"] = self.options.with_openjpeg
 
-        cmake.configure(build_folder=self._build_subfolder)
-        return cmake
+        tc.generate()
+        cd = CMakeDeps(self)
+        cd.generate()
 
     def build(self):
-        self._patch_sources()
-
-        cmake = self._configure_cmake()
+        apply_conandata_patches(self)
+        cmake = CMake(self)
+        cmake.configure()
         cmake.build()
 
     def package(self):
-        cmake = self._configure_cmake()
+        copy(self, "LICENSE.md", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
```

Comment:
```suggestion
        copy(self, "LICENSE*.md", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
```

We still need to copy the old BSD license